lib/Traits/TraitsClasses/ArcGisMapServerCatalogItemTraits.ts

```diff
@@ -57,4 +57,20 @@ export default class ArcGisMapServerCatalogItemTraits extends mixTraits(
       "date range when layer in time-enabled."
   })
   maxRefreshIntervals: number = 1000;
+
+  @primitiveTrait({
+    name: "Time Window Duration",
+    description:
+      "Specify a time window duration from the current time that this dataset is available. Positive: forward; negative: backward.",
+    type: "number"
+  })
+  timeWindowDuration?: number;
+
+  @primitiveTrait({
+    name: "Time Unit",
+    description:
+      "The time unit for the `Time Interval`. Any units supported by `moment` module are valid, such as, `year`, `month`, `week`, `day`, `hour`, etc.",
+    type: "string"
+  })
+  timeUnit?: string;
```

Comment:
Ideally I would prefer if it was possible to merge `timeUnit` and `timeWindowDuration` into a single trait. But otherwise, please rename this as `timeWindowUnit` to match the other trait name.

Reply:
If merging number and unit into a single trait like `timeWindowDuration: "1d"`, we still need an extra trait indicating forward or backward window. My approach is to use positive or negative sign for that purpose.

Reply:
Ok, let's make them all separate traits then, i.e `timeWindow`, `timeWindowUnits` and `timeWindowDirection`.

Reasoning:
- It aligns more with the original ArcGis parameters. It also requires less thought from the user when configuring, like how do I map forward/backward attribute to a `timeWindow` value etc.
- In the future, when using an editor UI to configure it is easier to enter a `number` and then pick the units and direction from drop-downs.